denizen_scripts/survival/repo-link/MobScripts/SlimeStuff.dsc

```diff
@@ -26,21 +26,24 @@ slime_damage_handler:
 #Check for empty slot or invalid item
       - if !<player.equipment_map.contains[<[equipment]>]> || !<player.equipment_map.get[<[equipment]>].ends_with[_helmet]>:
         - actionbar "<&e>Acid splashes on you, burning your <[body_part]>."
+#Damage player without armor
         - determine <context.damager.mythicmob.level.mul[5]>
+      - else:
 #Define variables
-      - define item <player.equipment_map.get[<[equipment]>]>
-      - define Durrbillty <[item].durability.add[<context.damager.mythicmob.level.sub.[6].mul[<context.damager.size.div[2].round_up>]>]>
+        - define item <player.equipment_map.get[<[equipment]>]>
+        - define Durrbillty <[item].durability.add[<context.damager.mythicmob.level.sub.[6].mul[<context.damager.size.div[2].round_up>]>]>
 #Item break check
-      - if <[item].repairable>:
-        - if <[Durrbillty]> >= <[item].max_durability>:
-          - playeffect effect:ITEM_CRACK at:<player.location.above[0.5].forward[0.4]> special_data:<[item]> offset:0.2 quantity:15
-          - playsound <context.location> sound:ENTITY_ITEM_BREAK
-          - take slot:<[slot]>
+        - if <[item].repairable>:
+          - if <[Durrbillty]> >= <[item].max_durability>:
+            - playeffect effect:ITEM_CRACK at:<player.location.above[0.5].forward[0.4]> special_data:<[item]> offset:0.2 quantity:15
+            - playsound <context.location> sound:ENTITY_ITEM_BREAK
+            - take slot:<[slot]>
+            - ex narrate "Your <[slot]> has melted from the acid!" target:<player>
```

Comment:
this narrate:
- is default going to narrate to `<player>`, we don't need `target:<player>`
- is kind of odd to tell the player that their "(37/38/39/40) has melted from the acid" - maybe we could specify the item instead?

Reply:
Thats a good point, I meant for it to say `helmet|chestplate|boots|leggings`

Reply:
this is my initial suggestion until we can in-depth map what item we'd like to narrate
```suggestion
            - ex narrate "Your armor has melted from the acid!"
```